packages/account/src/Sections/Security/DerivPassword/change-password-form.jsx

```diff
@@ -52,13 +52,13 @@ class ChangePasswordForm extends React.Component {
         });
 
         if (values.new_password) {
-            if (!/^[ -~]{6,25}$/.test(values.new_password)) {
-                errors.new_password = localize('Password length should be between 6 to 25 characters.');
+            if (!/^[ -~]{8,25}$/.test(values.new_password)) {
```

Comment:
why are we testing regex here and not just length?

Reply:
I believe this is leftover from when this was our only password validation. I have made this validation function leaner now. Thanks.